CONTRIBUTING.md

```diff
@@ -19,6 +19,21 @@ As contributors and maintainers to this project, you are expected to abide by cu
 conduct. More information can be found at:
 [Contributor Code of Conduct](https://docs.rapids.ai/resources/conduct/).
 
+## Documentation contributions
+
+To make a contribution only to cuDF documentation:
+
+1. Create the `cudf_dev` conda environment following the steps in [Building with a conda environment](#building-with-a-conda-environment).
+2. Activate the environment and install a recent version of `cudf`
+
+```bash
+conda activate cudf_dev
+mamba install cudf -c rapidsai-nightly -c conda-forge
```

Comment:
Since conda uses the fast solver now, we could change `mamba` to `conda`, but I don't know how much we still use `mamba` throughout the docs.

Reply:
```suggestion
conda install cudf -c rapidsai-nightly -c conda-forge
```